homework-submission/myapp/src/Data.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+class Data extends React.Component {
+
+    render() {
+        return (
+            <ul className='todos'>
```

Comment:
Why are you using `<ul>`? however it doesn't have `<li>`? You can just use `<div>`, or you can use `<li>` here and outside this component wrap all the `<Data />` components with `<ul>`